codemods/jscodeshift/next/13/comment-deletable-files/index.ts

```diff
@@ -0,0 +1,36 @@
+import type { API, FileInfo, Options, Transform } from 'jscodeshift';
+import { basename } from 'node:path';
+
+export default function transform(file: FileInfo, api: API, _: Options) {
+	const baseName = basename(file.path);
+
+	if (
+		!baseName.startsWith('_document') &&
+		!baseName.startsWith('_app') &&
+		!baseName.startsWith('_error')
+	) {
+		return undefined;
+	}
+
+	const { j } = api;
+
+	const root = j(file.source);
+
+	root.find(j.Program).forEach((programPath) => {
+		const program = programPath.value;
+
+		const comments = program.comments ?? [];
+
+		comments.push(
+			j.commentBlock(
+				'This file should be deleted. Please migrate its contents to appropriate files',
```

Comment:
Can you add a link to https://beta.nextjs.org/docs/upgrade-guide#migrating-_documentjs-and-_appjs?